grepper_python/exceptions.py

```diff
@@ -0,0 +1,45 @@
+"""
+grepper-python.exceptions
+~~~~~~~~~~~~~~~~~~~
+This module contains the set of grepper-python's exceptions.
+"""
+
+
+# 400
+class BadRequest:
```

Comment:
I would recommend having your exeptions stem from the `Exception` class so you can do `raise BadRequest` (or whatever)

```suggestion
class BadRequest(Exception):
```